content/en/01-about-pixie/02-data-sources.md

```diff
@@ -34,9 +34,9 @@ Pixie automatically traces the following protocols:
 | Protocol      | Support             | Notes                          |
 | :------------ | :------------------ | :----------------------------- |
 | HTTP          | Supported           |                                |
-| HTTP2/gRPC    | Partially Supported | Currently only for Golang apps |
+| HTTP2/gRPC    | Partially Supported | Currently only for Golang apps with [debug information](/reference/admin/debug-info). |
 | DNS           | Supported           |                                |
-| NATS          | Supported           | Requires a NATS build with debug symbols (default in nats-server v2.3.4+). |
+| NATS          | Supported           | Requires a NATS build with [debug information](/reference/admin/debug-info) (default in nats-server v2.3.4+). |
```

Comment:
While debug symbols are default, debug info is not :(

So we should probably remote the "(default in ...)"